tests/network/localnet/conftest.py

```diff
@@ -2,27 +2,36 @@
 
 import pytest
 from ocp_resources.namespace import Namespace
+from ocp_resources.node import Node
 
 import tests.network.libs.nodenetworkconfigurationpolicy as libnncp
 from libs.net.traffic_generator import Client, Server
-from libs.net.vmspec import IP_ADDRESS, lookup_iface_status
 from libs.vm.vm import BaseVirtualMachine
 from tests.network.libs import cluster_user_defined_network as libcudn
-from tests.network.localnet.liblocalnet import LOCALNET_TEST_LABEL, NETWORK_NAME, localnet_cudn, localnet_vm
+from tests.network.localnet.liblocalnet import (
+    LOCALNET_TEST_LABEL,
+    create_traffic_client,
+    create_traffic_server,
+    localnet_cudn,
+    localnet_vm,
+    run_vms,
+)
 from utilities.constants import (
     WORKER_NODE_LABEL_KEY,
 )
 from utilities.infra import create_ns
 
-_IPERF_SERVER_PORT = 5201
+LOCALNET_BR_EX_NETWORK = "localnet-br-ex-network"
+LOCALNET_OVS_BRIDGE_NETWORK = "localnet-ovs-network"
+NNCP_INTERFACE_TYPE_OVS_BRIDGE = "ovs-bridge"
```

Comment:
1. Why the `NNCP_` prefix?
2. There is actually no need to define this string constant, you can and should import it from [utilities/constants](https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/utilities/constants.py#L140).

Reply:
The constant and it's naming are here to have a meaningful name. If you think that this name is not clear or meaningful enough, I'll be happy to refactor it.

Reply:
I'm not sure how meaningful this name is, but OTOH - you (and others) might find this naming clear and meaningful, while I (and different others :) ) may not. So I'm leaving this one for you as well to decide whether you choose to change it or not.
I'm fine with whatever you decide> Don't forget to resolve this conversation too.